### DISCUSSION TITLE
Stable Diffusion - GifToGif

### DISCUSSION THREAD

**Discussion by saddam213** [saddam213/OnnxStack#56] [Ideas]
Was having a play around with an idea @TheyCallMeHex came up with recently

StableDiffusion on frames of a GIF image, this process would be prompt control only, no control net so its limited, however initial results have been kinda cool

| Input `Prompt: Obama` | Output |
| :--- | :--- |
![Source](https://github.com/saddam213/OnnxStack/assets/4353684/d9da90e4-d97c-4f32-b4c5-47bf7e2d0786) |  ![Result](https://github.com/saddam213/OnnxStack/assets/4353684/214e4be9-5b7f-4e60-9474-ec8c20bc2095) |

| Input `Prompt: Elon Musk` | Output |
| :--- | :--- |
![Source](https://github.com/saddam213/OnnxStack/assets/4353684/1f56aa06-bc9b-4b66-bd4b-4ce32b3769fe) | ![Result](https://github.com/saddam213/OnnxStack/assets/4353684/326b7564-3bfa-4c01-b711-d6e32dedb0e3) |

| Input `Prompt: Mikasa Ackerman`  | Output |
| :--- | :--- |
![Source](https://github.com/saddam213/OnnxStack/assets/4353684/30109389-8597-4c68-9c6d-913b190a18d8) | ![Result](https://github.com/saddam213/OnnxStack/assets/4353684/132f0d91-2a6d-4652-847f-a206c50a5495) |

This initial process is available in the `OnnxStack.Console` app now, working on UI intergration

**Comment by saddam213** [comment]
| Input `Prompt: Elon Musk dancing in an office`  | Output |
| :--- | :--- |
![Source](https://github.com/saddam213/OnnxStack/assets/4353684/a1c2e612-476b-4b78-b7a5-d3581430e951) | ![Result](https://github.com/saddam213/OnnxStack/assets/4353684/11e17a35-1322-4ecb-8995-b2e3771161a6)

**Comment by TheyCallMeHex** [comment]
| Input `Prompt: trotting unicorn, bright fantastic colors`  | Output |
| :--- | :--- |
![source](https://github.com/saddam213/OnnxStack/assets/90013272/c447e14d-96c6-4ed7-93c9-e618ce3fdb4b) | ![Result](https://github.com/saddam213/OnnxStack/assets/90013272/5ba4343d-d6fd-4273-8b52-9aa7cd78df96) |
